### ISSUE TITLE
`E (780) cam_hal: cam_dma_config(300): frame buffer malloc failedCamera init failed with error 0xffffffff`

### ISSUE THREAD

**Issue by mrmonteith** [Xinyuan-LilyGO/T-CameraPlus-S3#4]
### IDE and version or firmware

Ardino IDE 2.3.2

### Operating System

Windows 10

### Description

I get a `E (780) cam_hal: cam_dma_config(300): frame buffer malloc failedCamera init failed with error 0xffffffff`

### Sketch

```cpp
* @Description(EN):
 *      This is a routine that projects the OV2640 camera data onto the T-CameraPlus-S3 screen.
 *
 * @version: V1.0.0
 * @Author: LILYGO_L
 * @Date: 2023-11-15 16:57:21
 * @LastEditors: LILYGO_L
 * @LastEditTime: 2023-11-25 09:04:35
 * @License: GPL 3.0
```

### Debug Message

```plain
na
```

### Other steps to Reproduce

No

**Comment by Llgok** [comment]
@mrmonteith 
The issue has already been addressed in point #3. You can refer to the solution method in point #3.

**Comment by Llgok** [comment]
Closed due to inactivity for a long period.
